lib/internal/Magento/Framework/View/Helper/SecureHtmlRenderer.php

```diff
@@ -119,8 +119,12 @@ function {$listenerFunction} () {
                         targetElement = event.target;
                     }
                     {$listenerFunction}.apply(targetElement);
-                }
+                    if ({$elementName}.nodeName == 'A' &&  "$eventName" == 'onclick') {
+                        event.preventDefault();
+                    }
+		}
             }
+	
 script;
 
         return $this->renderTag('script', ['type' => 'text/javascript'], $script, false);
```

Comment:
Hard-code logic check nodename is not good way
You should add change in https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Backend/Block/Widget/Grid/Column/Renderer/Action.php#L193

Reply:
CC @rogyar

Reply:
Example 
![Screenshot from 2020-11-26 06-57-55](https://user-images.githubusercontent.com/1908873/100293048-f8387380-2fb4-11eb-8b7b-d78d90a2bdab.png)

Reply:
thanks for the suggestion, but the preventDefault only works if called within function onclick into: 

https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/View/Helper/SecureHtmlRenderer.php#L116

the issue is resolved by just adding onclick if condition but I have put a double-check for the anchor tag. 
let me know if I need to remove the anchor tag check in this.